charts/partials/templates/_network-validator.tpl

```diff
@@ -3,10 +3,14 @@ name: linkerd-network-validator
 image: {{.Values.proxy.image.name}}:{{.Values.proxy.image.version | default .Values.linkerdVersion }}
 imagePullPolicy: {{.Values.proxy.image.pullPolicy | default .Values.imagePullPolicy}}
 securityContext:
-  runAsUser: 65534
+  allowPrivilegeEscalation: false
```

Comment:
Not against to specifying this, but wouldn't it be false by default?

Reply:
It's required. Otherwise the RS fails with this error:
```
Warning  FailedCreate  5s (x3 over 13s)  replicaset-controller  (combined from similar events): Error creating: pods "linkerd-proxy-injector-f458577d5-pds5j" is forbidden: violates PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "linkerd-network-validator" must set securityContext.allowPrivilegeEscalation=false)
```
Maybe required because it's not always false by default, but depends on the value of other fields:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#securitycontext-v1-core